app/scripts/lib/mascara-proxy-provider.js

```diff
@@ -0,0 +1,51 @@
+module.exports = function setupMascaraProxyProvider (extensionProvider) {
+  let isMascaraActive = false
+  if (global.mascaraProvider) {
```

Comment:
sure i will change it but... food for thought: what if the race condition shows up it wont be obvious till you send a tx that mascara isnt global. Left this way it throws an error if you expected it to be their and it's not.